purchase_order_product_recommendation/wizards/purchase_order_recommendation.py

```diff
@@ -0,0 +1,290 @@
+# Copyright 2019 David Vidal <david.vidal@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.addons import decimal_precision as dp
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+from datetime import datetime, timedelta
+
+
+class PurchaseOrderRecommendation(models.TransientModel):
+    _name = 'purchase.order.recommendation'
+    _description = 'Recommended products for current purchase order'
+
+    order_id = fields.Many2one(
+        'purchase.order',
+        'Purchase Order',
+        default=lambda self: self._default_order_id(),
+        required=True,
+        readonly=True,
```

Comment:
@chienandalu ??
```
        required=True,
        readonly=True
```

Reply:
Its an OCA convention (https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#pep8-options):

> When making a comma separated list, dict, tuple, ... with one element per line, append a comma to the last element. This makes it so the next element added only changes one line in the changeset instead of changing the last element to simply add a comma.

Reply:
@chienandalu Wrong Topic

My question is `required & readonly` are True then how user put a value, in fields not any domain right now?

Reply:
The wizard is invoked from the order itself and that's where the id comes from. When a user calls the wizard from a PO is setting the value already.

Reply:
@chienandalu Sounds Good